src/main/java/com/google/javascript/clutz/Depgraph.java

```diff
@@ -38,7 +39,8 @@ Depgraph withNonrootsAsRoots() {
     Depgraph res = new Depgraph();
     res.roots.addAll(roots);
     res.roots.addAll(nonroots);
-    res.externs.addAll(externs);
+    res.rootExterns.addAll(rootExterns);
+    res.nonrootExterns.addAll(nonrootExterns);
```

Comment:
is this correct? it's oddly assymmetric.

Reply:
@rkirov 

I've looked over the code, and currently nothing depends on the state of `rootExterns` or `nonrootExterns` after `withNonrootsAsRoots()` is called.  I agree with Martin that the less surprising state of affairs in the future is that all the externs are in `rootExterns`.

Reply:
yep, should be `res.rootExterns.addAll(nonrootExterns)'. This branch code is only executed when 'strictDeps' is off, and we usually have it on.